.travis.yml

```diff
@@ -35,8 +35,11 @@ before_script:
   - if [[ "$DB" == "mysql" ]]; then mysql -e "create database phpcr_tests;"; fi
   - if [[ "$DB" == "pgsql" ]]; then psql -c "create database phpcr_tests;" -U postgres; fi
   - php tests/generate_phpunit_config.php $DB
+  - cp $DB.phpunit.xml phpunit.xml
```

Comment:
The benchmarking bootstrap (`.phpbench`) reads `phpunit.xml` ten `phpunit.xml.dist`.